src/test/java/uk/gov/justice/hmpps/prison/dsl/Alias.kt

```diff
@@ -21,12 +21,12 @@ class AliasBuilderRepository(
     lastName: String,
     firstName: String,
     birthDate: LocalDate,
-  ): AliasId = offenderRepository.findOffenderByNomsId(offenderId.offenderNo).orElseThrow().let {
+  ): AliasId = offenderRepository.findRootOffenderByNomsId(offenderId.offenderNo).orElseThrow().let {
     offenderRepository.save(
       Offender.builder()
         .nomsId(it.nomsId)
-        .rootOffenderId(it.rootOffenderId)
-        .rootOffender(it.rootOffender)
+        .rootOffenderId(it.id)
```

Comment:
it wasn't clear what was going on previously, but since we're already getting the `rootOffender` then can use `it`